applications/openshift/networking/routes_rate_limit/tests/have_routes_rate_limit.pass.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# remediation = none
+
+yum install -y jq
+
+kube_apipath="/kubernetes-api-resources"
+
+mkdir -p "$kube_apipath/apis/route.openshift.io/v1"
+
+routes_apipath="apis/route.openshift.io/v1/routes?limit=500"
+
+cat <<EOF > "$kube_apipath/$routes_apipath"
+{
+    "apiVersion": "v1",
+    "items": [
+        {
+            "apiVersion": "route.openshift.io/v1",
+            "kind": "Route",
+            "metadata": {
```

Comment:
shouldn't this fail because this doesn't have the annotation?

Reply:
Because we skipped all the openshift related namespaces

Reply:
aaaah! Gotcha